colortest

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
-theme=$(dirname $0)/scripts/${1:-base16-default-dark.sh}
+
+if [[ -z $BASE16_SHELL_THEME_NAME_PATH ]]; then
+  BASE16_SHELL_THEME_NAME_PATH="${XDG_CONFIG_HOME:-$HOME/.config}/tinted-theming/theme_name" 
+fi
+read current_theme_name < "$BASE16_SHELL_THEME_NAME_PATH"
```

Comment:
How does this react if the file doesn't exist? I'm not super familiar with this code.

Reply:
If the file doesn’t exist then the variable set will be empty and the ternary type thing below it is used in. Perhaps I could change the name of “current_theme_name” to something else since it may not exist.

Reply:
I renamed to `current_theme_name_from_config_file` and added comments